examples/cluster/plot_adjusted_for_chance_measures.py

```diff
@@ -55,10 +54,15 @@ def uniform_labelings_scores(score_func, n_samples, n_clusters_range,
             scores[i, j] = score_func(labels_a, labels_b)
     return scores
 
+
+def ami_score(U, V):
+    return metrics.adjusted_mutual_info_score(U, V,
+                                              average_method='arithmetic')
+
 score_funcs = [
     metrics.adjusted_rand_score,
     metrics.v_measure_score,
-    metrics.adjusted_mutual_info_score,
+    ami_score,
```

Comment:
maybe,
```py
from functools import partial

score_funcs = [
    ...   
    partial(metrics.adjusted_mutual_info_score, average_method='arithmetic')
    ...
    ]
```

Reply:
I had it as a partial first, but then later the example uses the `__name__` of it which is not set for partials. First I had it set manually, then Joel suggested `def`ing it to have it more readable for the users.

Reply:
OK, I see.